### DISCUSSION TITLE
Invalid time zone specified: Etc/Unknown reported in the UI

### DISCUSSION THREAD

**Discussion by jeremy-watttime** [dagster-io/dagster#18709] [Q&A] [answered]
Hello, I am evaluating dagster by running locally and keep getting this error in the UI.  The postgres db is in UTC and my computer is local time.
I have not found a place where I am supposed to configure the time zone or otherwise fix this error.

> Sorry, graph can't be displayed.
> Please report this error to the Dagster team via GitHub or Slack. Refresh the page to try again.
> Invalid time zone specified: Etc/Unknown
> RangeError: Invalid time zone specified: Etc/Unknown
>     at Date.toLocaleDateString (<anonymous>)
>     at timestampToString (http://127.0.0.1:3000/_next/static/chunks/418.4b907660b16d2f78.js:1:451423)
>     at TimestampDisplay (http://127.0.0.1:3000/_next/static/chunks/418.4b907660b16d2f78.js:1:1165975)
>     at Xh (http://127.0.0.1:3000/_next/static/chunks/framework-d6095a5336d3e425.js:9:60991)
>     at x (http://127.0.0.1:3000/_next/static/chunks/framework-d6095a5336d3e425.js:9:119446)
>     at Vk (http://127.0.0.1:3000/_next/static/chunks/framework-d6095a5336d3e425.js:9:99131)
>     at http://127.0.0.1:3000/_next/static/chunks/framework-d6095a5336d3e425.js:9:98998
>     at Jk (http://127.0.0.1:3000/_next/static/chunks/framework-d6095a5336d3e425.js:9:99005)
>     at Ok (http://127.0.0.1:3000/_next/static/chunks/framework-d6095a5336d3e425.js:9:95761)
>     at Hk (http://127.0.0.1:3000/_next/static/chunks/framework-d6095a5336d3e425.js:9:94317)

I am running dagster version '1.5.12'

**Comment by hellendag** [comment]
Hi @jeremy-watttime, thanks for reporting this.

- What OS and browser (including version) are you using?
- Does the bug also occur in other browsers?
- What is your actual correct local timezone?
- If you run `Intl.DateTimeFormat().resolvedOptions().timeZone` in your browser console, what value does it show?

**Reply by jeremy-watttime** [reply]
- Ubuntu 18.04.6 LTS  Chrome 116.0.5845.179
- Firefox 113.0.2 seems to be ok
- America/New_York
- 'Etc/Unknown'

**Reply by jeremy-watttime** [reply, marked as answer]
After a closing/reopening Chrome, it seems to work and `Intl.DateTimeFormat().resolvedOptions().timeZone` in the Chrome console returns "America/New_York"
